corehq/apps/hqmedia/tasks.py

```diff
@@ -227,3 +217,51 @@ def build_application_zip(include_multimedia_files, include_index_files, app,
         )
 
     DownloadBase.set_progress(build_application_zip, 100, 100)
+
+
+def _find_missing_locale_ids_in_ccz(file_cache):
+    for file_path in ('default/app_strings.txt', 'suite.xml'):
+        if file_path not in file_cache:
+            return [_("Could not find {file_path} in CCZ").format(file_path)]
```

Comment:
This for loop would only return one error at a time, but all other code paths return all errors at once. Is it worth chaging this to allow 2 errors if neither file is found?

Reply:
Pretty unlikely for even one of these to happen, but yeah, good best practice. 05d0fe04cde